stories/module-adsense-settings.stories.js

```diff
@@ -58,16 +59,24 @@ const completeSettings = {
 	siteStatus: SITE_STATUS_ADDED,
 	accountSetupComplete: true,
 	siteSetupComplete: true,
+	webStoriesAdUnit: '0123456789',
+	webStoriesActive: true,
 };
 
 const Settings = createLegacySettingsWrapper( 'adsense' );
 
+const setUpAdUnits = ( registry ) => {
+	registry.dispatch( STORE_NAME ).receiveGetAdUnits( fixtures.adunits, { accountID: fixtures.accounts[ 0 ].id, clientID: fixtures.clients[ 0 ].id } );
+	registry.dispatch( STORE_NAME ).finishResolution( 'getAdUnits', [] );
```

Comment:
We need to provide `accountID` and `clientID` arguments to properly finish resolution:

```suggestion
	const accountID = fixtures.accounts[ 0 ].id;
	const clientID = fixtures.clients[ 0 ].id;
	registry.dispatch( STORE_NAME ).receiveGetAdUnits( fixtures.adunits, { accountID, clientID } );
	registry.dispatch( STORE_NAME ).finishResolution( 'getAdUnits', [ accountID, clientID ] );
```